core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/rules/SnowflakeCallMapper.scala

```diff
@@ -63,6 +65,14 @@ class SnowflakeCallMapper extends ir.CallMapper with ir.IRHelpers {
       case ir.CallFunction("POSITION", args) => ir.CallFunction("LOCATE", args)
       case ir.CallFunction("REGEXP_LIKE", args) => ir.RLike(args.head, args(1))
       case ir.CallFunction("REGEXP_SUBSTR", args) => regexpExtract(args)
+      case ir.CallFunction("SEQ1", args) =>
+        // TODO: Warn users about The seq[1/2/4/8] function in Snowflake and monotonically_increasing_id in Databricks
+        //  are equivalent in logic,
+        //  but the output size generated by seq[1/2/4/8] can be different. Please be cautious while using.
```

Comment:
We could/should truncate and cast the `MonotonicallyIncreasingId` to the proper width (1, 2, 4 or 8 bytes).

Reply:
If the optional sign argument is 0, the sequence continues at 0 after wrap-around. If the optional sign argument is 1, the sequence continues at the smallest representable number based on the given integer width.

Reply:
I have removed the SEQ from this PR. I will create a new PR for the same